### DISCUSSION TITLE
PocketBase + SurrealDB

### DISCUSSION THREAD

**Discussion by compgeniuses** [pocketbase/pocketbase#571] [Q&A] [answered]
I noticed that Pocketbase is focused on an embeded style database architecture and is currently using sqlite, however sqlite is not that much scalable.
i came accross this wonderful databse https://surrealdb.com/ which might be great to help in pocketbase

**Comment by ganigeorgiev** [comment, marked as answer]
PocketBase is not designed to scale horizontally out of the box and that's OK (see why it was created in [Presentator #183](https://github.com/presentator/presentator/issues/183)).

SurrealDB looks very interesting, but it is completely different project and BaaS on its own, similar to Supabase, Nhost, Appwrite, etc.

If you need scaling and high availability out of the box, I recommend considering some of the above mentioned open source solutions.
